src/main/index.ts

```diff
@@ -103,14 +103,16 @@ app.on("ready", async () => {
   }
 
   extensionLoader.init();
-
   extensionDiscovery.init();
   windowManager = WindowManager.getInstance<WindowManager>(proxyPort);
 
   // call after windowManager to see splash earlier
   try {
     const extensions = await extensionDiscovery.load();
 
+    // Start watching after bundled extensions are loaded
+    extensionDiscovery.watchExtensions();
```

Comment:
I don't see how this is broken, watchExtensions contains `await this.whenLoaded;` which is set at the end of `load`

Reply:
We can probably revert this, I did separate these just to understand the current flow better.